src/shapes/group.class.js

```diff
@@ -123,12 +123,9 @@
           center = this.getCenterPoint();
 
       object.set({
-        originalLeft: objectLeft,
```

Comment:
.originalLeft/.originalRight property not used in all library. Why do we store it?